Ginger/Ginger/Run/RunSetActions/RunSetActionDeliveryMethodConfigPage.xaml.cs

```diff
@@ -0,0 +1,83 @@
+﻿using GingerCore.GeneralLib;
+using System.Windows;
+using System.Windows.Controls;
+using GingerCore;
+namespace Ginger.Run.RunSetActions
+{
+    /// <summary>
+    /// Interaction logic for RunSetActionDeliveryMethodConfigPage.xaml
+    /// </summary>
+    public partial class RunSetActionDeliveryMethodConfigPage : Page
+    {
+        public RunSetActionDeliveryMethodConfigPage(RunSetActionSendSMS runSetActionSendSMS)
+        {
```

Comment:
All the bindings are with Email fields. so we can have only 1 constructor and pass Email object to it. Also context should not be null

Reply:
Changed.